File: examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairCodePaseCommand.java
package com.matter.controller.commands.pairing;

import chip.devicecontroller.ChipDeviceController;
import com.matter.controller.commands.common.CredentialsIssuer;

public final class PairCodePaseCommand extends PairingCommand {
  public PairCodePaseCommand(ChipDeviceController controller, CredentialsIssuer credsIssue) {
    super(
        controller,
        "code-paseonly",
        PairingModeType.CODE_PASE_ONLY,
        PairingNetworkType.NONE,
        credsIssue);
  }

  @Override
  protected void runCommand() {}
}

File: examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairCodePaseCommand.kt
/*
 * SPDX-FileCopyrightText: 2023 Project CHIP Authors
 *
 * SPDX-License-Identifier: Apache-2.0
 *
 */
package com.matter.controller.commands.pairing

import chip.devicecontroller.ChipDeviceController
import com.matter.controller.commands.common.CredentialsIssuer

class PairCodePaseCommand(controller: ChipDeviceController, credsIssue: CredentialsIssuer?) :
  PairingCommand(
    controller,
    "code-paseonly",
    credsIssue,
    PairingModeType.CODE_PASE_ONLY,
    PairingNetworkType.NONE
  ) {
  override fun runCommand() {}
}
